### PR TITLE
Fixes OperationVariable's internal representation of SMC & Entity

### PR DESCRIPTION
**Pull request by FrankSchnicke**

### CODE REVIEW

**Comment by FrankSchnicke** [comment]
@alexgordtop please review

File: src/main/java/org/eclipse/basyx/submodel/metamodel/map/submodelelement/operation/OperationVariable.java [hunk 1]
```diff
@@ -114,7 +119,33 @@ public void setValue(ISubmodelElement value) {
...
+			return value;
+		}
+	}
+
+	private Object handleSMC(ISubmodelElement value) {
+		return SubmodelElementMapCollectionConverter.smElementToMap((SubmodelElementCollection) value);
+	}
+
+	private Object handleEntity(Entity value) {
+		Collection<ISubmodelElement> statements = value.getStatements();
```
Comment (alexgordtop) [inline src/main/java/org/eclipse/basyx/submodel/metamodel/map/submodelelement/operation/OperationVariable.java:144]: @FrankSchnicke As far as I correctly understand the SubmodelElementMapCollectionConverter, Entities are processed there, too. Why do you split here?

Comment (FrankSchnicke) [reply src/main/java/org/eclipse/basyx/submodel/metamodel/map/submodelelement/operation/OperationVariable.java]: Thanks for pointing this out - I updated the PR.

**Comment by alexgordtop** [comment]
Ok, nice, then from my point of view: Go :)

**Review by Daespen — APPROVED** [review]

**Pull request merged**